scripts/push-to-quay

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+if [[ -z "$3" ]]; then
+  echo "Usage: $0 [NAMESPACE] [REPOSITORY] [RELEASE]"
+  exit 1
+fi
+
+if [[ -z "$QUAY_USERNAME" ]]; then
+  echo -n "Quay Username: "
+    read QUAY_USERNAME
+fi
+
+if [[ -z "$QUAY_PASSWORD" ]]; then
+  echo -n "Quay Password: "
+  read QUAY_PASSWORD
+  echo
+fi
+
+TOKEN=$(curl -sH "Content-Type: application/json" -XPOST https://quay.io/cnr/api/v1/users/login -d '
+{
+    "user": {
+        "username": "'"${QUAY_USERNAME}"'",
+        "password": "'"${QUAY_PASSWORD}"'"
+    }
+}' | jq -r '.token')
+
+
+NAMESPACE=$1
+REPOSITORY=$2
+RELEASE=$3
+
+function cleanup() {
+    rm -f ${REPOSITORY}.tar.gz
+}
+
+trap cleanup EXIT
+
+tar czf ${REPOSITORY}.tar.gz -C "../${NAMESPACE}/bundles/${REPOSITORY}" bundle.${RELEASE}.yaml
+
+BUNDLE=$(cat ${REPOSITORY}.tar.gz | base64 -w 0)
+
+curl -H "Content-Type: application/json" \
+     -H "Authorization: ${TOKEN}" \
+     -XPOST https://quay.io/cnr/api/v1/packages/${NAMESPACE}/${REPOSITORY} -d '
```

Comment:
When you are pushing to a new repository, it gets created as private by default. Please look into making it public here, otherwise it is a manual step that needs to be done after pushing.

Reply:
Spoke with @aravindhp offline and got the OK to add this feature as an enhancement in a later date.

Reply:
@awgreene `csrf_token` is used to prevent CSRF and is used by the browser. However, when calling the API directly `csrf_token` is not required. You can change visibility of a repository by calling the following API - https://docs.quay.io/api/swagger/#!/repository/changeRepoVisibility. Passing the authorization token should be enough.